src/main/java/com/oltpbenchmark/benchmarks/auctionmark/AuctionMarkLoader.java

```diff
@@ -40,6 +40,8 @@
 import java.util.concurrent.TimeUnit;
 import java.util.regex.Pattern;
 
+import static com.oltpbenchmark.types.DatabaseType.ORACLE;
```

Comment:
```suggestion
import com.oltpbenchmark.types.DatabaseType;
```

and reference `DatabaseType.ORACLE` below would be more consistent with other parts of the code.